NodeBase/entry_point.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 source /opt/bin/functions.sh
+/opt/bin/generate_config > /opt/selenium/config.json
```

Comment:
Before the commit 0dd193a (*Moving config generation in docker files*) the line was here.

I believe that it's because NodeBase is not runned directly, it's used in other images that have generate_config.

---

For instance for Chrome.
In NodeBase [Dockerfile.txt](https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeBase/Dockerfile.txt#L56), the entry point is set :
```
CMD ["/opt/bin/entry_point.sh"]
```
It calls `entry_point.sh` which calls for `generate_config`.

In NodeChrome [Dockerfile.txt](https://github.com/SeleniumHQ/docker-selenium/blob/master/NodeChrome/Dockerfile.txt#L36), the `generate_config` file is copyied inside the image, before the entrypoint is called :
```
COPY generate_config /opt/bin/generate_config
```